ios/MullvadVPN.xcodeproj/project.pbxproj

```diff
@@ -800,6 +813,7 @@
 				CODE_SIGN_ENTITLEMENTS = PacketTunnel/PacketTunnel.entitlements;
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = G7CDBEG477;
+				ENABLE_BITCODE = NO;
```

Comment:
FYI had to disable bitcode because I had some issues with Go and compiling it with bitcode on.